src/MCT.js

```diff
@@ -20,72 +20,194 @@
  * at runtime from the About dialog for additional information.
  *****************************************************************************/
 /* eslint-disable no-undef */
-define([
-  'EventEmitter',
-  './api/api',
-  './api/overlays/OverlayAPI',
-  './api/tooltips/ToolTipAPI',
-  './selection/Selection',
-  './plugins/plugins',
-  './ui/registries/ViewRegistry',
-  './plugins/imagery/plugin',
-  './ui/registries/InspectorViewRegistry',
-  './ui/registries/ToolbarRegistry',
-  './ui/router/ApplicationRouter',
-  './ui/router/Browse',
-  './ui/layout/AppLayout.vue',
-  './ui/preview/plugin',
-  './api/Branding',
-  './plugins/licenses/plugin',
-  './plugins/remove/plugin',
-  './plugins/move/plugin',
-  './plugins/linkAction/plugin',
-  './plugins/duplicate/plugin',
-  './plugins/importFromJSONAction/plugin',
-  './plugins/exportAsJSONAction/plugin',
-  'vue'
-], function (
-  EventEmitter,
-  api,
-  OverlayAPI,
-  ToolTipAPI,
-  Selection,
-  plugins,
-  ViewRegistry,
-  ImageryPlugin,
-  InspectorViewRegistry,
-  ToolbarRegistry,
-  ApplicationRouter,
-  Browse,
-  Layout,
-  PreviewPlugin,
-  BrandingAPI,
-  LicensesPlugin,
-  RemoveActionPlugin,
-  MoveActionPlugin,
-  LinkActionPlugin,
-  DuplicateActionPlugin,
-  ImportFromJSONAction,
-  ExportAsJSONAction,
-  Vue
-) {
+import EventEmitter from 'EventEmitter';
+import { createApp, markRaw } from 'vue/dist/vue.esm-bundler';
```

Comment:
```suggestion
import { createApp, markRaw } from 'vue';
```

Reply:
That was a hacky fix - Vue throws some sort of error when that isn't used (something about `createApp` being imported wrongly)

Reply:
@LeoDog896 I think we can fix this just by adding a webpack alias to `.webpack/webpack.common.js`:

`vue: 'vue/dist/vue.esm-bundler'`

Reply:
That could work 👍